app/routers/authentication.py

```diff
@@ -5,15 +5,11 @@
 from fastapi.responses import Response
 from sqlalchemy.orm.session import Session
 from .. import schemas, database
-from app.registration import reg_env
 from datetime import timedelta
-from ..auth.token import (
-    create_access_token,
-    validate_access_token,
-    ACCESS_TOKEN_EXPIRE_MINUTES,
-)
+from ..auth.token import create_access_token, validate_access_token
```

Comment:
can we please provide the absolute imports - replace the `..auth` import with `toplevelmodule_here.auth`